src/api/app/views/webui/search/_results.html.haml

```diff
@@ -1,25 +1,26 @@
-#search-results
-  - @results.each do |result|
-    - if result.kind_of? Project
-      - project = result.name
-      - package = nil
-      - rtype = 'project'
-    - else
-      - project = result.project_name
-      - package = result.name
-      - rtype = 'package'
-    .search_result
-      %h6.data-title
-        = sprite_tag(rtype, class: rtype, title: rtype.humanize)
-        = project_or_package_link project: project, package: package, short: false, trim_to: nil
-        = content_tag :span, result.sphinx_attributes, style: 'display:none'
+.mt-4#search-results
+  %h4 Results:
+  - results.each do |result|
+    :ruby
+      if result.is_a? Project
```

Comment:
Shouldn't we use `duck typing` instead of static class types for branching? Something like ```ruby
if result.respond_to?(:project_name)
```

Reply:
I agree. And this code can be changed after this pull request. But the goal of this pull request is only replacing Bento code with the already existing Bootstrap code.

Definitely a review in this direction should be done for the whole application. I counted 124 ocurrences of  `is_a?` in the actual code inside `src/api/app/`:

```bash
> grep -r is_a? src/api/app/ | wc -l
124
>
```

Reply:
Ok, I see